internal/rbd/nodeserver.go

```diff
@@ -421,7 +420,7 @@ func (ns *NodeServer) stageTransaction(
 	transaction.isStagePathCreated = true
 
 	// nodeStage Path
-	readOnly, err = ns.mountVolumeToStagePath(ctx, req, staticVol, stagingTargetPath, devicePath)
+	_, err = ns.mountVolumeToStagePath(ctx, req, staticVol, stagingTargetPath, devicePath)
```

Comment:
remove extra readOnly return from mountVolumeToStagePath just return err.

Reply:
sure, removed it.. thanks @Madhu-1 .. ptal.